testsuite/tests/typing-safe-linking/Makefile

```diff
@@ -3,6 +3,13 @@
 SOURCES = a.ml b_bad.ml
 OBJECTS = $(SOURCES:%.ml=%.cmo)
 
+default:
+	@if $(SAFE_STRING) ; then \
+	  echo " ... testing 'b_bad.ml' => skipped" ; \
```

Comment:
Does that mean that if OCaml is configured with `-safe-string`, therefore, that it's still worth running this test, just with the bottom line of both `a.ml` and `b_bad.ml` chopped?